tests/unit/commands/local/lib/test_sam_layer_provider.py

```diff
@@ -72,34 +74,29 @@ class TestSamLayerProvider(TestCase):
     }
 
     def setUp(self):
+        Path("PyLayer").mkdir(exist_ok=True)
```

Comment:
Unit tests should not be creating resources on the system. We should mock out the call instead of this.

As an additional note: When we do create things on the system, they should always be done in in a tmp directory (through tempfile)